test/commands/scanner/run.severity.test.ts

```diff
@@ -0,0 +1,114 @@
+import {expect} from '@salesforce/command/lib/test';
+import {setupCommandTest} from '../../TestUtils';
+import {Messages} from '@salesforce/core';
+import path = require('path');
+
+
+Messages.importMessagesDirectory(__dirname);
+const runMessages = Messages.loadMessages('@salesforce/sfdx-scanner', 'run');
+
+describe('scanner:run', function () {
+	this.timeout(10000); // TODO why do we get timeouts at the default of 5000?  What is so expensive here?
```

Comment:
We might not need this timeout since these tests are much simpler.

Reply:
Removed!